src/main/java/lotto/view/OutputView.java

```diff
@@ -70,12 +76,20 @@ private static String winningStatisticsMessage(StatisticsOfLottos statisticsOfLo
 
         Arrays.stream(Rank.values())
                 .filter(rank -> rank != MISS)
-                .forEach(rank -> winningStatisticsMessageBuilder.append(String.format(WINNING_STATISTICS_MESSAGE, rank.matchCount(), rank.winningMoney(), statisticsOfLottos.numberOfMatchCount(rank)))
+                .forEach(rank -> winningStatisticsMessageBuilder.append(String.format(winningStaticsMessageFormat(rank), rank.matchCount(), rank.winningMoney(), statisticsOfLottos.numberOfMatchCount(rank)))
                         .append(NEXT_LINE));
 
         return winningStatisticsMessageBuilder.toString();
     }
 
+    private static String winningStaticsMessageFormat(Rank rank) {
+        if(rank == SECOND) {
```

Comment:
Nit: 객체에게 메시지를 보내서 처리해보면 어떨까요?
```suggestion
        if (rank.isSecond()) {
```